capa/v29.3.0/README.md

```diff
@@ -3,3 +3,20 @@
 ## Changes compared to v29.2.0
 
 This release does not contain any changes to components or apps, but makes use of an updated machine image, which includes a fix for accessing private Elastic Container Registries (ECR).
+
+#### ⚠️ Breaking change introduced in v29.2.0 with `cluster-aws` version 2.1.0
+
+- Do not allow additional properties in the following fields in order to avoid unnoticed typos:
+  - global.connectivity.network
+  - global.connectivity.network.pods
+  - global.connectivity.network.services
+  - global.connectivity.subnets[]
+  - global.connectivity.topology
+  - global.controlPlane
+  - global.controlPlane.additionalSecurityGroups[]
+  - global.controlPlane.machineHealthCheck
+  - global.controlPlane.oidc
+  - global.providerSpecific
+  - global.providerSpecific.instanceMetadataOptions
```

Comment:
```suggestion
- Do not allow additional properties in the following fields in order to avoid unnoticed typos:

  - `global.connectivity.network`
  - `global.connectivity.network.pods`
  - `global.connectivity.network.services`
  - `global.connectivity.subnets[]`
  - `global.connectivity.topology`
  - `global.controlPlane`
  - `global.controlPlane.additionalSecurityGroups[]`
  - `global.controlPlane.machineHealthCheck`
  - `global.controlPlane.oidc`
  - `global.providerSpecific`
  - `global.providerSpecific.instanceMetadataOptions`
```

(from Marco's suggestion)